client/src/components/qa/AnswerList.jsx

```diff
@@ -1,16 +1,59 @@
-import React from 'react';
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
 import Answer from './Answer.jsx';
 
-function AnswerList({ list }) { // list is an object of objects
+function AnswerList({ answerList, getQuestions }) {
+  const [answerCount, setAnswerCount] = useState(2);
+  const [moreAnswers, setMoreAnswers] = useState('See More Answers');
+
   const ansArr = [];
-  Object.keys(list).forEach((key) => {
-    ansArr.push(list[key]);
+  Object.keys(answerList).forEach((key) => {
+    ansArr.push(answerList[key]);
   });
+  ansArr.sort((a, b) => b.helpfulness - a.helpfulness);
+
+  const handleMoreAnswers = (e) => {
+    e.preventDefault();
+    if (answerCount === 2) {
+      setAnswerCount(ansArr.length);
+      setMoreAnswers('Collapse Answers');
+    } else if (answerCount === ansArr.length) {
+      setAnswerCount(2);
+      setMoreAnswers('See More Answers');
+    }
+  };
+
+  if (ansArr.length > 2) {
```

Comment:
If you want, it looks like you can move the conditional inside of return statement to render the Answer Component unconditionally, and then render form on condition, if that makes sense (i.e. place the conditional on line 40 and get rid of the second return statement)

Reply:
Yeah that definitely makes sense, I tend to get a little lazy when it comes to conditional rendering so that would be a nice, more concise solution